package/yast2-kdump.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Mon Jun 26 15:27:06 CEST 2017 - schubi@suse.de
+
+- Crash in AutoYaST clone: Fix for "dup" while getting
+  crash_xen_kernel_values and crash_kernel_values values.
+  (bnc#1045965)
+- 3.3.0
```

Comment:
Jumping to 3.3.X in master? I guess it makes sense now that we branched SP3 and master will be SLE15. But, has it been discussed/agreed?